lib/cheet.rb

```diff
@@ -0,0 +1,30 @@
+class Cheet 
+
+  attr_reader :db_conn
+  
+  def initialize
+    if ENV['RACK_ENV'] == 'test'
+      @db_conn = PG.connect(dbname: 'chitter_test')
+    else 
+      @db_conn = PG.connect(dbname: 'chitter')
+    end 
+  end 
+
+  def all 
+    # currently returning arrays, would possibly change to passing instances ?  
+    arr = []
+    result = @db_conn.exec("SELECT * FROM cheets")
+    result.each do |post| 
+      arr << [post["posts"], post["date"], post["time"]]
+    end
+    arr
```

Comment:
Maybe look at potentially changing/refactoring using #each to #map. This way you could remove the arr variable as #map would return a new array for you